.github/workflows/ci.yaml

```diff
@@ -20,16 +20,14 @@ jobs:
     steps:
       # Checks-out your repository under $GITHUB_WORKSPACE, so your job can access it
       - uses: actions/checkout@v2
-      - name: Mount bazel action cache
+      - name: Mount bazel caches
         uses: actions/cache@v2
         with:
-          path: "~/.cache/bazel"
-          key: bazel
-      - name: Mount bazel repo cache
-        uses: actions/cache@v2
-        with:
-          path: "~/.cache/bazel-repo"
-          key: bazel-repo
+          path: |
+            "~/.cache/bazel"
+            "~/.cache/bazel-repo"
+          key: bazel-cache-${{ hashFiles('**/BUILD.bazel', '**/*.bzl', 'WORKSPACE') }}
```

Comment:
why do we need a cache key here? can't we always restore the cache regardless of the hashes of certain files, and rely on bazel to ignore outdated entries in it?

Reply:
As we discussed offline, once a cache is stored it can never be updated, so we need to write under new cache keys whenever things change. See comment I added for clarification.